src/main/java/bio/terra/service/common/gcs/GcsUriUtils.java

```diff
@@ -102,21 +100,22 @@ public static String getControlPath(
     return lastWildcard >= 0 ? newPath.substring(0, lastWildcard + 1) : newPath;
   }
 
-  public static String makeHttpsFromGs(String gspath) {
-    try {
-      BlobId locator = GcsUriUtils.parseBlobUri(gspath);
-      String gsBucket = locator.getBucket();
-      String gsPath = locator.getName();
-      String encodedPath =
-          URLEncoder.encode(gsPath, StandardCharsets.UTF_8.toString())
-              // Google does not recognize the + characters that are produced from spaces by the
-              // URLEncoder.encode
-              // method. As a result, these must be converted to %2B.
-              .replaceAll("\\+", "%20");
-      return String.format(
-          "https://www.googleapis.com/storage/v1/b/%s/o/%s?alt=media", gsBucket, encodedPath);
-    } catch (UnsupportedEncodingException ex) {
```

Comment:
Just curious, are we no longer concerned about hitting this exception?

Reply:
Exactly.  I changed how we're getting the encoding so it doesn't throw anymore